test/pylib/scylla_cluster.py

```diff
@@ -707,6 +714,49 @@ async def _mark_dirty(self, request) -> aiohttp.web.Response:
         self.cluster.is_dirty = True
         return aiohttp.web.Response(text="OK")
 
+    async def _server_stop(self, request: aiohttp.web.Request, gracefully: bool) \
+                        -> aiohttp.web.Response:
+        """Stop a server. No-op if already stopped."""
+        assert self.cluster
+        ret = await self.cluster.server_stop(request.match_info['id'], gracefully)
+        return aiohttp.web.Response(status=200 if ret[0] else 500, text=ret[1])
+
+    async def _cluster_server_stop(self, request) -> aiohttp.web.Response:
+        """Stop a specified server"""
+        assert self.cluster
+        return await self._server_stop(request, gracefully = False)
+
+    async def _cluster_server_stop_gracefully(self, request) -> aiohttp.web.Response:
+        """Stop a specified server gracefully"""
+        assert self.cluster
+        return await self._server_stop(request, gracefully = True)
+
+    async def _cluster_server_start(self, request) -> aiohttp.web.Response:
+        """Start a specified server (must be stopped)"""
+        assert self.cluster
+        ret = await self.cluster.server_start(request.match_info['id'])
+        return aiohttp.web.Response(status=200 if ret[0] else 500, text=ret[1])
+
+    async def _cluster_server_restart(self, request) -> aiohttp.web.Response:
+        """Restart a specified server (must be already started)"""
+        assert self.cluster
+        ret = await self.cluster.server_restart(request.match_info['id'])
+        return aiohttp.web.Response(status=200 if ret[0] else 500, text=ret[1])
+
+    async def _cluster_server_add(self, request) -> aiohttp.web.Response:
+        """Add a new server"""
+        assert self.cluster
+        server_id = await self.cluster.add_server()
+        return aiohttp.web.Response(text=server_id)
+
+    async def _cluster_server_remove(self, request) -> aiohttp.web.Response:
+        """Remove a specified server"""
```

Comment:
Would be nice to see, besides the issue of what this waits for, an comment on what is "removing" a server and how it differs from "stopping" it. I have a pretty good idea, but I think it should be in writing.